client/www/components/dash/explorer/RecentlyDeletedAttrs.tsx

```diff
@@ -0,0 +1,146 @@
+import { Button, Dialog, SectionHeading, useDialog } from '@/components/ui';
+import config from '@/lib/config';
+import { SchemaNamespace, InstantApp, DBAttr } from '@/lib/types';
+import { useDashFetch } from '@/lib/hooks/useDashFetch';
+import useSWR from 'swr';
+import { errorToast } from '@/lib/toast';
+import { InstantReactWebDatabase } from '@instantdb/react';
+import { ClockIcon } from '@heroicons/react/24/outline';
+import { ArrowUturnLeftIcon } from '@heroicons/react/24/solid';
+import { add, formatDistanceToNow } from 'date-fns';
+import { useEffect } from 'react';
+
+const getNamesByNamespace = (attr: DBAttr): Record<string, string> => {
```

Comment:
nit: maybe let's call this `softDeletedAttr`, to make it clear this is really only for soft-deleted attrs